elements/rh-cta/rh-cta.ts

```diff
@@ -122,7 +126,7 @@ export class RhCta extends LitElement {
 
   protected override async getUpdateComplete(): Promise<boolean> {
     if (this.icon) {
-      await import('@patternfly/elements/pf-icon/pf-icon.js');
+      await import('@rhds/elements/rh-icon/rh-icon.js');
```

Comment:
This is superfluous if we import statically as above

I believe it was Zack who requested we dynamically import, for performance reasons, and I tend to agree

Reply:
removed static import.